src/core/p5.Renderer.js

```diff
@@ -97,6 +98,30 @@ class Renderer {
     this._pushPopDepth--;
     Object.assign(this.states, this._pushPopStack.pop());
     this.updateShapeVertexProperties();
+    this.updateShapeProperties();
+  }
+
+  bezierOrder(order) {
```

Comment:
Are we going to be using setters as getters when no arguments are passed? 

I'm not sure if there are best practices around this type of thing, but I can see some pros and cons. Having separate, explicit setters and getters is probably significantly better for readability, but it also significantly increases the size of the API. For functions such as these, the getter case will be used much less often by users, so it may make sense to avoid adding separate getter functions. It also may make sense since none of these functions has "set" in the name, and I doubt there's enough reason to change the names.

If we incorporate getter functionality into setters, I guess we'd want to update the reference so that it documents the return value. And I guess we could make the setters return the value that's passed in, if an argument is provided. Also, for consistency, it seems like we'd want to do this with all setters including e.g. `stroke()`. As long as we're consistent, I probably wouldn't worry too much, although having to increase the size of the documentation for a feature users will rarely use seems less than ideal. Maybe the question is whether we need the getter functionality at all, or if we can work around it.

Right now, we're just getting things working, but it seems like this might warrant public discussion at some point, if we think we might use setters as getters, as done here. What do you think?

Reply:
Cool. I'll add this to the task list: Make all shape setters function as getters, and update their documentation.

Would it make sense to always return a value? So, if an argument is provided, then that value would be returned? The benefit is simplicity. The return value always has the same type, and its documentation is simple: the return value is always the current setting. Otherwise, the reference would need to say that the return value is undefined if an argument is passed, but it's the value of the current setting if no argument is passed, which sounds more complicated.

Reply:
Decision: we'll resolve this in another PR.